linode/manager
# feat: [M3-6993] – Add VPC data to Linode Detail header

## Description 📝
For Linodes assigned to a VPC, display the linkified VPC label, the subnets list, and the VPC IPv4 address in a "VPC" section in the Linode Detail header.

## To-Do:
- [x] Minor code cleanup

## Major Changes 🔄
For Linodes assigned to a VPC, display a "VPC" section in the Linode Detail header.

Also, add an `enabled` argument to the `useVPCsQuery()` so that the query doesn't run on Linode Detail pages if the VPC section shouldn't be displayed.

## Preview 📷
![Screenshot 2023-09-07 at 2 17 54 PM](https://github.com/linode/manager/assets/114682940/8e513fe5-e33e-48b5-8eb2-ed0944087a90)

## How to test 🧪
Confirm that a Linode assigned to a VPC has the VPC section added to its Detail header.

Confirm that a Linode not assigned to a VPC does not have the VPC section in its Detail header.
